Fixes #5230 - Update Gatwick (EGKK) SMR - Rapid Exit Taxiway changes

olivercesarczyk opened the pull request:
Fixes #5230 

# Summary of changes

Add ER RET, add E TWY WIP, remove E, add temporary E RET in red and change hold labels to yellow as on real SMR display.
Additionally a small change to the apron area to match reference images.

# Screenshots (if necessary)
Initial proposal:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/0d829ed2-be25-4a7f-892c-8b1fefbaf31f)
ARIAC 2403 chart:
![2403newER](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/0a0b6cae-f4f1-4502-b022-e6c356c5279d)
Final SMR:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/5e860cdf-275c-450f-a666-476e81db364c)

PLM1995 reviewed (changes requested):
This is what the issue called for:

>Mark E as closed, but leave in place, suggest using a `X` or subtly changing the colour. In practice bear in mind most sceneries will not change for a while so we expect it to still be used. Leave the current label and line in place at E.

Bright red is not subtle, and you have changed E's shape rather than leaving it in place. You also appear to have removed the hold line.

Some hold labels seem to have disappeared.

The WIP area does not need adding yet, this will only confuse people, it serves no purpose to us on VATSIM until it's implemented, it's only charted so IRL pilots don't drive onto it after vacating ER I would have thought.

You will need to make a changelog entry and update the .kmz as well.

Ideally comments don't need the " - " after the `;`, I usually do a find and replace to sort this.

Please check the issue number in the PR title.

In future please put a comment on the issue saying you're going to work on it, or simply assign yourself. Saves duplication of effort :)

olivercesarczyk commented:
I have changed how E is displayed within the layers, so when it gets totally removed, only one region needs to be removed. I can extend it further to the runway like before if you wish, although there is now a strip removed between it and the runway. I will also change the colour to the pier 6 WIP colour and see what it looks like.

FR and ER hold labels are not on the real SMR display.

The new E taxiway in red between J and 08L is fully complete and ready for use, but will not be commissioned just yet. I have also changed the layers around in this area, so when it will be opened only one region needs to be removed and no other changes will have to be made. I dont see why people might be confused about it being there if it is red, charted and completed.

I will update the .kmz once we have agreed on the final look of the SMR.

olivercesarczyk commented:
Updated SMR with changes:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/f53ee9b9-9017-4263-a029-cfa70945c62f)
New E "link" taxiway constructed IRL:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/5eb6be91-5504-4fe5-9053-e67a75f39df3)

PLM1995 commented:
> I have changed how E is displayed within the layers, so when it gets totally removed, only one region needs to be removed.

Sounds good.

-----

> I will also change the colour to the Heathrow red colour and see what it looks like.

I feel red is slightly harsh given most aircraft will be using it to vacate for a while. I was thinking more a different colour of grey or something? I also think keep the hold line in. The hold letter being red is fine in my opinion, to remind controllers not to taxi outbounds to it.

-----

> FR and ER hold labels are not on the real SMR display.

Real SMRs don't have stand numbers either, but this is VATSIM, and we need to help controllers more, especially when we're implementing taxiway/turnoff changes at busy airports in my opinion. I do appreciate your yearning for realism but we do have to make some considerations for VATSIM.

-----

> The new E taxiway in red between J and 08L is fully complete and ready for use, but will not be commissioned just yet. I have also changed the layers around in this area, so when it will be opened only one region needs to be removed and no other changes will have to be made. I dont see why people might be confused about it being there if it is red, charted and completed.

I would be tempted to leave this off all together (could just comment it out since you've added it already) until it's at least commisioned, if not in scenery. Having it the same colour as the former RET E which will still be being used but it not being available would cause confusion in my opinion. So Mayyyybe you could leave it in in red and make E a different colour, but I don't see what it adds for us on VATSIM at this stage.

-----

> I will update the .kmz once we have agreed on the final look of the SMR.

I think the changes in the .kmz are probably correct and we're just talking about colours/commenting out areas, but obviously you'll need to upload it and edit the changelog before anyone can approve this PR.

-----

Thanks,
Peter

olivercesarczyk commented:
New changes:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/85702171/1f883b5c-4e14-49a7-b28f-e740af86c93b)

PLM1995 commented on _data/SMR Files/EGKK/Old KMZs/Gatwick SMR (moved@2403).kmz:
If we do decide this needs keeping (e.g. if you do end up renaming/reordering everything), I would suggest naming without the `moved@` for consistency.

PLM1995 commented on _data/SMR Files/EGKK/EGKK 2403.kmz:
I am confused by the 'unused' folder.
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/53f322ef-ad0b-4ec1-8b16-5f1a23be7bf5)
If those elements are not used in the .sct files, why are they being kept in the .kmz?

There are also quite a few items here
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/e029a409-c26b-4fd5-906e-fca88514957b)
which don't seem present on your screenshot or in the .txt files, I know E we'll be saving for when it's added in future, are the rest like that or could they be removed for avoid confusion?

PLM1995 reviewed (changes requested):
Screenshot of the ES output looks good and I'm happy with that :)

From what I can see, you seem to have renamed/re-ordered loads of stuff between the two .kmz versions, but these changes haven't been carried forward into the .sct .txt files. It is essential that naming etc. is consistent accross the data to allow for easy development in future. This Issue was for a very simple change to the E/ER RETs, if you want to rework/improve the whole SMR, that's great, but the names and ordering should match between the .kmz and the .sct .txt files please. I would usually expect that kind of major rejigging to be done in a separate PR so this AIRAC work doesn't get unduly delayed.

I don't know which is easier for you to sort at this stage, splitting the changes into this and a potential future PR, or getting everything in line with the updated .kmz? I'll be happy either way so whatever's easier for you :) If you do re-jig stuff, feel free to alphabetise the GEO things, atm it's a bit hard work to find stuff. But I know that's nothing to do with the Issue so please don't feel any obligation.

Hope I explained that clearly enough.

Thanks,
Peter
